File: WalletApp/src/app/modules/company/payload/CompanyUpdatePayload.ts

import {Employee} from "../../employee/model/business/employee";

export interface CompanyUpdatePayload {
  companyId: string;
  name: string;
  Address:string;
  Description:string;
  Is_manage:string;
  Active:boolean;
  Deleted:boolean;
  Deleted_by:boolean;
  employees: Employee[]
}

File: backend/src/main/java/ipeps/pwd/wallet/entity/payload/CompanyUpdatePayload.java
package ipeps.pwd.wallet.entity.payload;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.UUID;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class CompanyUpdatePayload {
    UUID company_id;
    String Title;
    String Description;
    String Address;
    String Is_managed;
    Boolean Active;
    Boolean Deleted;
    Boolean Deleted_by;


}
